src/assets/i18n/en.json5

```diff
@@ -5386,6 +5386,24 @@
 
   "admin.system-wide-alert.title": "System-wide Alerts",
 
+  "search.advanced.filters.head": "Advanced Search",
+  "filter.search.operator.placeholder": "Operator",
+  "filter.search.text.placeholder": "Search text",
+  "filter.search.placeholder": "Select filter",
+  "search.filters.operator.equals.text": "Equals",
+  "search.filters.operator.notequals.text": "Not Equals",
+  "search.filters.operator.notcontains.text": "Not Contains",
+  "search.filters.operator.contains.text": "Contains",
+  "search.filters.filter.title.text": "Title",
+  "search.filters.filter.author.text": "Author",
+  "search.filters.filter.subject.text": "Subject",
```

Comment:
I believe all the keys above this line are from a different PR (#2608). So, I think they can be safely removed from this PR (but please double check that I'm correct)